docker/Dockerfile

```diff
@@ -1,62 +1,24 @@
 FROM continuumio/miniconda3
 
-MAINTAINER Gilles Bodart <gillesbodart@users.noreply.github.com>
+# Install build-essential (compiler and development tools)
```

Comment:
Why did you remove the `MAINTAINER` field? I don't know if it is actually needed, but I guess we need to make sure about this.

Reply:
Oh, My apologies. It was a mistake while making multiple updates to the file.
I’ll add it now.